packages/composites/anchor/README.md

```diff
@@ -9,6 +9,7 @@ The adapter takes the following environment variables:
 | Required? |                Name                |                         Description                         |                                             Options                                             |                  Defaults to                   |
 | :-------: | :--------------------------------: | :---------------------------------------------------------: | :---------------------------------------------------------------------------------------------: | :--------------------------------------------: |
 |    ✅     |         `ETHEREUM_RPC_URL`         |                The URL to Ethereum RPC node                 |                                                                                                 |                                                |
+|           |        `ETHEREUM_CHAIN_ID`         |               The blockchain id to connect to               |                                                                                                 |                       1                        |
```

Comment:
Just for consistency with how this is referred to in general. Needs to be updated in all places.

```suggestion
|           |        `ETHEREUM_CHAIN_ID`         |               The chain id to connect to               |                                                                                                 |                       1                        |
```